executor/executor_test.go

```diff
@@ -1022,6 +1022,50 @@ func (s *testSuite) TestJSON(c *C) {
 	result.Check(testkit.Rows(`3 {} <nil>`))
 }
 
+func (s *testSuite) TestGeneratedColumnDDL(c *C) {
```

Comment:
We can put tests to ddl_db_test.go, otherwise, the unit test coverage is too low.